cert/spec.go

```diff
@@ -49,6 +49,11 @@ type SpecOptions struct {
 	// invocations.
 	ServiceManagerName string `json:"svcmgr" yaml:"svcmgr"`
 
+	// DisregardActionTargetStatus if set to false, disables reload/restart attempts
```

Comment:
Looks like a missed comment when renaming the variable

Reply:
Ack and I'll fix- the naming of this configurable has frankly been hell.  Suggestions welcome.